src/js/API-categories.js

```diff
@@ -0,0 +1,14 @@
+export const fetchCategories = async () => {
+  const API_URL = 'https://tasty-treats-backend.p.goit.global/api';
+
+  try {
+    const response = await fetch(`${API_URL}/categories`);
+    const apiData = await response.json();
+    const categories = apiData.map(category => category.name);
+
+    return categories;
+  } catch (error) {
+    console.error('ERROR', error);
+    return [];
+  }
```

Comment:
Use than-catch in the outermost code execution, remove error catch in a function